<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoveProduto</class>
 <widget class="QDialog" name="RemoveProduto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Remover Pedido</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Confirmar_bt">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>190</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Id:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Id_txt">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome Funcionario:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Nome_txt">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Produto:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Qtd:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Qtd_txt">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Produto_txt">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
